### ISSUE TITLE
MenuItem does not active on Enter keypress

### ISSUE THREAD

**Issue by mimarz** [equinor/design-system#1702]
When testing the `Menu` component with keyboard navigation, the selected menu item did not close or active on keypress. This is in violation of how expected behaviour for menu should be according to [aria best practices](https://www.w3.org/TR/wai-aria-practices-1.1/#menu)
